libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -85,6 +86,7 @@
     private final ExecutorService threadPool = Executors.newFixedThreadPool(1);
 	private SmartStore smartStore;
 	private RestClient restClient;
+    private SyncManagerLogger logger;
```

Comment:
SyncManager and sync targets do all their logging through a SyncManagerLogger.
One can set the logging level desired (by default INFO, but tests run with DEBUG). At DEBUG, key informations (e.g. requests / responses etc) are printed out which makes it easy to follow the whole sync operation.